packages/babel-preset-react-server/index.js

```diff
@@ -1,19 +1,19 @@
-module.exports = {
-  plugins: [
-    require('babel-plugin-transform-es2015-arrow-functions'),
-    require('babel-plugin-transform-es2015-block-scoping'),
-    require('babel-plugin-transform-es2015-classes'),
-    require('babel-plugin-transform-es2015-computed-properties'),
-    require('babel-plugin-transform-es2015-constants'),
-    require('babel-plugin-transform-es2015-destructuring'),
-    require('babel-plugin-transform-es2015-modules-commonjs'),
-    require('babel-plugin-transform-es2015-parameters'),
-    require('babel-plugin-transform-es2015-shorthand-properties'),
-    require('babel-plugin-transform-es2015-spread'),
-    require('babel-plugin-transform-es2015-template-literals'),
-    require('babel-plugin-transform-object-rest-spread'),
-  ],
-  presets: [
-    require('babel-preset-react'),
-  ],
+module.exports	=	{
```

Comment:
Oh, it's another set of mid-line tabs.  Weird that it renders differently in the PR's diff view vs the conversation view.